src/resources/other/BusolaExtensions.routes.js

```diff
@@ -38,32 +50,68 @@ const ColumnWrapper = ({ details }) => {
     if (layout) {
       setLayoutColumn(initialLayoutState);
     }
-  }, [layout, namespace, name]); // eslint-disable-line react-hooks/exhaustive-deps
+  }, [layout, isColumnLeyoutEnabled, namespace, name]); // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
```suggestion
  }, [layout, isColumnLeyoutEnabled, namespace, name]);
```

Reply:
Nope, we have `initialLayoutState` and `setLayoutColumn` there